doc/usage.rst

```diff
@@ -361,42 +361,42 @@ to reproduce the version text file.
 Building Mac OS X App Bundles
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you specify only ``--onefile`` under Mac OS X, the output
-in :file:`dist` is a UNIX executable
-:file:`myscript`.
-It can be executed from a Terminal command line.
-Standard input and output work as normal through the Terminal window.
-
-If you also specify ``--windowed``, the ``dist`` folder contains
-two outputs: the UNIX executable :file:`myscript`
-and also an OS X application named :file:`myscript.app`.
+Under Mac OS X, |PyInstaller| always builds a UNIX executable in
+:file:`dist`.
+If you specify ``--onedir``, the output is a folder named :file:`myscript`
+containing supporting files and an executable named :file:`myscript`.
+If you specify ``--onefile``, the output is a single UNIX executable
+named :file:`myscript`.
+Either executable can be started from a Terminal command line.
+Standard input and output work as normal through that Terminal window.
+
+If you specify ``--windowed`` with either option, the ``dist`` folder
+also contains an OS X application named :file:`myscript.app`.
 
 As you probably know, an application is a special type of folder.
-The one built by |PyInstaller| contains a folder always named :file:`Contents`.
-It contains:
+The one built by |PyInstaller| contains a folder always named
+:file:`Contents` which contains:
 
   + A folder :file:`Frameworks` which is empty.
-  + A folder :file:`MacOS` that contains a copy of the same
-    :file:`myscript` UNIX executable.
   + A folder :file:`Resources` that contains an icon file.
   + A file :file:`Info.plist` that describes the app.
+  + A folder :file:`MacOS` that contains the same contents as
+    the ``--onedir`` bundle.
```

Comment:
Maybe append  "… (see above)"?

Reply:
I will change this wording. It is not important to know that `dist/myscript.app/Contents/MacOS/*` is a redundant copy of `dist/myscript/*` Also not something we want to promise in the doc as someday it might not be that way.